website/content/docs/operators/configuration/config-intro.md

```diff
@@ -1,17 +1,18 @@
 ---
-title: Intro to Heron Configuration
+title: Intro to Heron Internal Configuration
 ---
 
-Heron can be configured at two levels:
+Heron internal configuration can be configured at two levels:
```

Comment:
For "internal", I mean it is config applies to the whole cluster, and is transparent to topology developers (it is impossible for them to override it)